doc/nrf/dev_model_and_contributions/code_base.rst

```diff
@@ -218,6 +218,11 @@ In the case of OSS repositories, the git tag format reuses the upstream project'
 
 In this format, ``X``, ``Y`` and ``Z`` are the ``MAJOR``, ``MINOR`` and ``PATCH`` versions of the upstream project, and ``-ncsN`` is used to identify the number of |NCS| releases based on that particular version.
 
+In the case of |NCS| Add-ons, the git tag format might be different from the |NCS| version, that add-on is based on.
+Along with the version, it also has an optional ``--add-on`` suffix::
+
+  vX.Y.Z(-add-on)(-rcM)
```

Comment:
@FilipZajdel , do we need this mention of add-ons here?

Reply:
@ktaborowski Does every add-on installation follow the standard NCS installation? If yes, should this table be also updated? https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/installation/install_ncs.html#get_the_nrf_connect_sdk_code

You might also want to check the pages edited in this PR https://github.com/nrfconnect/sdk-nrf/pull/19880/files to see where else this tag format should be mentioned or linked to.

Reply:
I commented on Confluence [here](https://nordicsemi.atlassian.net/wiki/spaces/NC/pages/457376636/nRF+Connect+SDK+Add-Ons+-+Contributor+Guidelines?focusedCommentId=1478954499)